vcstools/rm_synth.py

```diff
@@ -398,21 +401,55 @@ def rmclean(self,niter=1000,gain=0.1,cutoff=2.,mask=False,verbose=True):
 	def print_rmstats(self):
 		"""
 		Print some stats about the results
-		
+
 		Some basic statistics will be reported to the terminal:
 			mean of RM clean components,
 		        dispersion of RM clean components
 
 		"""
 
 		if self.rmclean_done:
-			cp = where(abs(self.rm_comps)>0.)
-			#cabs = abs(self.rm_comps[cp])
-			#crm = self.phi[cp]
 			cabs = abs(self.rm_comps)
 			crm = self.phi.copy()
 			mcrm = sum(cabs*crm)/sum(cabs)
 			crmdisp = sqrt(sum(cabs*(crm-mcrm)**2)/sum(cabs))
 			print('Weighted mean RM of clean components is %f'%mcrm)
 			print('Weighted RM dispersion of clean components is %f'%crmdisp)
 
+
+def read_rmsynth_out(filename):
```

Comment:
Remove this function from here. This code wasn't written by us so it shouldn't be altered like this

Reply:
Alright but it needs to be put in the vcstools module (needs to be importable) so what do you want me to call the module file?

Reply:
probably just something like rmsynth_utils.py is fine. Whatever you reckon

Reply:
Done